modules/apps/transfer/keeper/msg_server.go

```diff
@@ -33,27 +29,6 @@ func (k Keeper) Transfer(goCtx context.Context, msg *types.MsgTransfer) (*types.
 		return nil, errorsmod.Wrapf(types.ErrSendDisabled, err.Error())
```

Comment:
This check was left in this function as the logic is not easy to implement on `OnSendPacket`, we would have to export `BankKeeper` but also add a lot of logic to convert `coins` back to `tokens`

Reply:
This check is necessary to be moved. We can move it into `keeper.OnSendPacket`. There's already logic to convert tokens to coin there

Reply:
Note: going forward, users can invoke `OnSendPacket` without using `MsgTransfer`, so it's unsafe to have any necessary checks remain in `MsgTransfer`